scripts/frontmatter/get-frontmatter.js

```diff
@@ -0,0 +1,93 @@
+const frontmatter = require('@github-docs/frontmatter');
+const { TYPES } = require('../constants');
+
+const GATSBY_CONTENT_TYPES = {
+  [TYPES.BASIC_PAGE]: 'page',
+  [TYPES.API_DOC]: 'apiDoc',
+  [TYPES.RELEASE_NOTE]: 'releaseNote',
+  [TYPES.RELEASE_NOTE_PLATFORM]: 'releaseNotePlatform',
+  [TYPES.TROUBLESHOOTING]: 'troubleshootingDoc',
+  [TYPES.WHATS_NEW]: 'nr1Announcement',
+};
+
+const GATSBY_TEMPLATE = {
+  [TYPES.BASIC_PAGE]: 'basicDoc',
+  [TYPES.API_DOC]: 'basicDoc',
+  [TYPES.RELEASE_NOTE]: 'basicDoc',
+  [TYPES.RELEASE_NOTE_PLATFORM]: 'basicDoc',
+  [TYPES.TROUBLESHOOTING]: 'basicDoc',
+  [TYPES.WHATS_NEW]: 'basicDoc',
+};
+
+const FRONTMATTER_TEMPLATE = {
+  [TYPES.BASIC_PAGE]: 'page',
+  [TYPES.API_DOC]: 'apiDoc',
+  [TYPES.RELEASE_NOTE]: 'releaseNote',
+  [TYPES.RELEASE_NOTE_PLATFORM]: 'releaseNotePlatform',
+  [TYPES.TROUBLESHOOTING]: 'troubleshooting',
+  [TYPES.WHATS_NEW]: 'whatsNew',
+};
+
+const getFrontmatter = (type, doc) => {
+  const defaultFrontmatter = {
+    title: doc.title.replace(':', '-'),
+    contentType: GATSBY_CONTENT_TYPES[type],
+    template: GATSBY_TEMPLATE[type],
+    topics: getTopics(doc),
+  };
+
+  return frontmatter.stringify('', {
+    ...defaultFrontmatter,
+    ...addCustomFrontmatter[FRONTMATTER_TEMPLATE[type]](doc),
+  });
+};
+
+const addCustomFrontmatter = {
+  apiDoc: (doc) => {
+    return;
+  },
+  page: (doc) => {
+    return {
+      japaneseVersion: doc.japaneseVersionExists === 'yes',
```

Comment:
I would expect that `japaneseVersion` would have a URL or some kind of semantic version associated with it by looking at the name. Since this looks like this returns a boolean, would it be better to name this something like `japaneseVersionExists` like the `doc` key is named?

Reply:
this is a tricky one. i think ideally we'd store the japanese version's URL if the version exists. so if `doc.japaneseVersionExists === 'yes'` we could take the URL path from `doc.docUrl` and prepend `https://docs.newrelic.co.jp/` to it

Reply:
@roadlittledawn will make that change!